src/info/langs/language.tera

```diff
@@ -107,6 +107,14 @@ impl Language {
             {% endfor %}
         }
     }
+
+    pub fn get_language_icon(&self) -> &'static str {
+        match self {
+            {% for language, attrs in languages -%}
+                Language::{{ language }} => "{{ attrs.icon | default(value='\u{25CF}') }}",
+            {% endfor %}
+        }
```

Comment:
Have this return `Option<&'static str>`. Where the icon is not defined, return `None`.

Then the caller can do `language.get_language_icon().unwrap_or("\u{25CF}")`.

Reply:
refactored it to return an Option.